src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: "NerdGraph tutorial: Topology for Applied Intelligence incident correlation"
+metaDescription: Use New Relic NerdGraph to set up or view Applied Intelligence topology, used for incident correlation.
+---
+
+This tutorial will explain how to use our NerdGraph API to set up your topology. Topology configuration allows your [Applied Intelligence decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) to better correlate incidents. 
+
+## Requirements [#requirements]
+
+Topology correlation is currently in limited release. See [topology requirements](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#topology-requirements).
+
+To use NerdGraph, you'll need a [user key](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#authentication). 
+
+## Overview of tutorial [#overview]
+
+With New Relic’s [Applied Intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/, you can create custom [decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) that govern how your incidents are correlated. One kind of custom decision logic uses the concept of “topology,” which in this context is a representation of your service map. 
+This doc will explain how to use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) to configure your topology mapping. 
+
+This tutorial will give example NerdGraph calls that: 
+* Set up your topology by creating vertices and edges
+* Delete vertices and edges
+* Retrieve topology data
+
+Before using NerdGraph to configure your topology, you should have a basic understanding of: 
+
+* [What Applied Intelligence decisions are](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) and [how topology correlation works](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#topology).
+* The topology structure you're trying to implement. One way to understand this is to use our [service map feature](/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/) to see how entities in your infrastructure relate to each other.  
+* [What NerdGraph is](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) and [how to use the NerdGraph API explorer](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#explorer) to run queries.
+
+## Mutation examples [#mutations]
+
+In NerdGraph, mutations are requests that perform an action ([learn more about NerdGraph terminology](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)). 
```

Comment:
Why not add this little bit extra: "such as creating a resource or changing a configuration."